From 11ef2bf99431fd3a8ededcb67bf03cb964ddad28 Mon Sep 17 00:00:00 2001
From: Fixture <fixture@example.com>
Date: Thu, 1 Jan 2026 01:00:00 +0000
Subject: Add whisper/1

---
 lib/toolkit.ex | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/toolkit.ex b/lib/toolkit.ex
index 269a776..d6adc76 100644
--- a/lib/toolkit.ex
+++ b/lib/toolkit.ex
@@ -6,4 +6,6 @@ defmodule Toolkit do
 
   @spec shout(String.t()) :: String.t()
   def shout(text), do: String.upcase(text) <> "!"
+  @spec whisper(String.t()) :: String.t()
+  def whisper(text), do: String.downcase(text) <> "..."
 end
